### ISSUE TITLE
Joseph Collett - Last Stop Solar // Unable to Submit Lead

### ISSUE THREAD

**Issue by BhargavRentala** [palmetto/Palmetto#329]
| Name | Email | Description | URL | Priority | WideSpread | StepstoReproduce |
|---|---|---|---|---|---|---|---|
| Maria Zmijewski | solar@laststopsolar.com | Cannot Submit Lead // Joseph Collett - Last Stop Solar | https://alchemy.palmetto.com/dashboard | Blocker | No | - Input lead information
- Submit utility information 
- Calculate successfully
- Submit "next" and continually loading |

**Comment by kellengarnaas** [comment]
I found a reference to RecordTypeId__c in the Lead Service, which I corrected to simply RecordTypeId. I am able to successfully create and push leads through to Opportunity conversion.

The example lead in this card is still broken due to the fact that an error neglected to populate the sf_lead_id in the database. The lead may need to be simply re-created from scratch and that should fix the issue.
